### PR TITLE
feat: card with different types

### PR DESCRIPTION
**Pull request by amarphule**
Created Card components with different types
1. Horizontal card with shadow
2. Horizontal card with dismiss
3. Verticle card
4. Verticle card with badge
5. Verticle card Text with an overlay
6. Card Text only

### CODE REVIEW

**Comment by spraveenofficial** [comment]
You can reduces the image sizes in cards, it's looking weird.

**Review by prathmeshjagtap — APPROVED** [review]

**Comment by amarphule** [comment]
> You can reduce the image sizes in cards, it's looking weird.

Done, Thank you for your suggetion.

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
+.p-1 {
+  padding: 1rem;
+}
+.pt-1 {
+  padding-top: 1rem;
+}
+.py-1 {
+  padding: 1rem 0px;
```
Comment (horsemaker) [inline components/card.css:8]: No need to write `0px`, only `0` would work.

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+.card {
+  width: 50%;
+  margin-bottom: 1rem;
+  border: 1px solid var(--secondary-color);
+  box-sizing: border-box;
+  position: relative;
+}
+.card-verticle {
```
Comment (horsemaker) [inline components/card.css:22]: Typo here. It should be `vertical` & not `verticle`.

File: components/card.css (hunk 1 above)
Comment (horsemaker) [inline components/card.css:8]: They feel like utility classes. Could we move them to utility in the next iteration?

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+}
+.card-shadow {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+}
+.card:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+.d-flex {
+  display: flex;
+}
```
Comment (horsemaker) [inline components/card.css:33]: This also feels like a utility class.

File: components/card.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+.py-1 {
+  padding: 1rem 0px;
+}
+
+.mb-2 {
+  margin-bottom: 2rem;
+}
+
+.card {
+  width: 50%;
```
Comment (nikhiljugale007) [inline components/card.css]: instead of using fixed width in % you can use min-width and max-width along with rems

**Comment by nikhiljugale007** [comment]
UI looking good,
One suggestion you can button in the horizontal card too.
Added one suggestion in code.
Rest look good.

**Comment by amarphule** [comment]
> UI looking good, One suggestion is you can button in the horizontal card too. Added one suggestion in code. The rest looks good.

Thank you for your suggestion. I kept cards simple for now with time I'll add buttons to them.

File: components/card.css (hunk 4 above)
Comment (amarphule) [reply components/card.css]: Thank you create the suggested change.

File: components/card.css (hunk 1 above)
Comment (amarphule) [reply components/card.css:8]: okay. Thank you

Comment (amarphule) [reply components/card.css:8]: yes we'll do that

File: components/card.css (hunk 3 above)
Comment (amarphule) [reply components/card.css:33]: yes sure

**Pull request merged**